Author: Andreas Beckmann <anbe@debian.org>
Description: dkms.conf updates
Forwarded: yes
Applied-Upstream: 2ae34dbd9f1ecb19de3e43d4b75008a078330e4b
--- v4l2loopback.orig/dkms.conf
+++ v4l2loopback/dkms.conf
@@ -1,6 +1,9 @@
 PACKAGE_NAME="v4l2loopback"
 PACKAGE_VERSION="0.14.0"
 
+# V4L2_COLORSPACE_DCI_P3
+BUILD_EXCLUSIVE_KERNEL_MIN="4.4"
+
 if [ -f $kernel_source_dir/.config ]; then
     . $kernel_source_dir/.config
     if ! { echo "$kernelver"; echo 5.18; } | sort -V -C; then
@@ -17,7 +20,7 @@
 
 # Items below here should not have to change with each driver version
 MAKE[0]="make KERNEL_DIR=${kernel_source_dir} v4l2loopback"
-CLEAN="make clean"
+CLEAN="make KERNEL_DIR=${kernel_source_dir} clean"
 
 BUILT_MODULE_NAME[0]="$PACKAGE_NAME"
 DEST_MODULE_LOCATION[0]="/extra"
